pkg/cmd/pr/checkout/checkout.go

```diff
@@ -144,11 +151,24 @@ func checkoutRun(opts *CheckoutOptions) error {
 		ref := fmt.Sprintf("refs/pull/%d/head", pr.Number)
 		if newBranchName == currentBranch {
 			// PR head matches currently checked out branch
-			cmdQueue = append(cmdQueue, []string{"git", "fetch", baseURLOrName, ref})
-			cmdQueue = append(cmdQueue, []string{"git", "merge", "--ff-only", "FETCH_HEAD"})
+
+			// If forced reset to remote
+			if opts.Force {
+				cmdQueue = append(cmdQueue, []string{"git", "fetch", baseURLOrName, ref})
+				cmdQueue = append(cmdQueue, []string{"git", "reset", "--hard", "FETCH_HEAD"})
+			} else {
+				cmdQueue = append(cmdQueue, []string{"git", "fetch", baseURLOrName, ref})
```

Comment:
Since the `git fetch` line is identical in both branches, could it be taken out of the `if`?